app/templates/code_conduct.html

```diff
@@ -1,76 +1,42 @@
 {% extends 'base_tabs.html' %}
 {% load static %}
-{% block head_title %}Code of conduct{% endblock %}
+{% block head_title %}Code of Conduct{% endblock %}
 
 {% block head %}
     <link rel="stylesheet" href="{% static 'css/main.css' %}">
 {% endblock %}
 
 {% block panel %}
+    <h1>{{ h_name }} Code of Conduct</h1>
+    <p>This is the Code of Conduct that we expect all HackUPC participants to abide. This is a customized version of the Code of Conduct used at MLH Member Hackathons. If you have a suggestion for this code of conduct or a question about them, please submit an issue <a href="https://github.com/hackupc/rules/issues" target="_blank">here</a>.</p>
```

Comment:
_to abide **by**_

Reply:
This should be changed on the code of conduct first